### DISCUSSION TITLE
Jump directly from one package body into another package body / header (Oracle)

### DISCUSSION THREAD

**Discussion by DoeWayne** [dbeaver/dbeaver#22402] [Ideas]
It would be great to get the possibility to jump from one oracle package body/header into another one, like you can do in SQL Developer or any other SQL tools.

Maybe it is possible in DBeaver, but did not found it.

**Comment by LonwoLonwo** [comment]
Hello @DoeWayne 

No, it is not implemented yet. Only the usual search function.

**Comment by DoeWayne** [comment]
will it come into the release sometimes?

**Reply by LonwoLonwo** [reply]
You need to create a [ticket](https://github.com/dbeaver/dbeaver/issues), then. Here are only discussions.
